python/src/FunctionalChaosSobolIndices_doc.i.in

```diff
@@ -162,13 +162,14 @@ marginalIndex : int
 Returns
 -------
 s : float
-    The first order Sobol' index."
+    The Sobol' first order index of an input variable or interaction index
+    of a group of input variables."
 
 // ---------------------------------------------------------------------
 
 %feature("docstring") OT::FunctionalChaosSobolIndices::getSobolTotalIndex
-"Get the total Sobol' index of an input variable or the interaction total
-index of a group of variables.
+"Get the Sobol' total index of an input variable or the total interaction
```

Comment:
I believe this phrase could lead to confusion. The interaction index is by definition not total in my opinion. Indeed, the total Sobol' index of a group of variables takes into account both direct and interaction effects of all members of the given variable group.

Reply:
These methods are based on the theory help page that I (deeply) updated recently. The _total interaction index of a group of variables_ is defined here :

http://openturns.github.io/openturns/latest/theory/reliability_sensitivity/sensitivity_sobol.html#interaction-sensitivity-indices-of-a-variable

Here is the relevant part of the page:

![image](https://github.com/openturns/openturns/assets/31351465/0edd6d62-ddd7-4604-b18b-e3ec38d2771c)

I hope that this remove any confusion in the text. I admit, however, that there are subtle differences between these indices: perhaps a talk on this new help page at the next TC will help. After that discussion, we may review the help page and see if any clarification may help.

Reply:
Do you agree on this point @JPelamatti ? If you do, I will resolve the discussion.